scripts/transact/mintLandsFromFile.ts

```diff
@@ -0,0 +1,68 @@
+import BN from 'bn.js';
+import fs from 'fs-extra';
+import {ethers, deployments} from 'hardhat';
+
+const networkName = 'rinkeby';
+const filePath = `tmp/${networkName}-landOwners.json`;
+const errorFilePath = 'tmp/mintLandErrors.json';
+
+(async () => {
+  const {execute} = deployments;
+  const errors = [];
+  type Land = {
+    coordinateX: BN;
+    coordinateY: BN;
+    size: BN;
+    tokenId: string;
+  };
+  type landOwnersMap = {[owner: string]: Land[]};
+
+  if (!fs.existsSync(filePath))
+    throw new Error(`file ${filePath} does not exist'`);
+  const landOwners: landOwnersMap = JSON.parse(
+    fs.readFileSync(filePath).toString()
+  );
+
+  const LandContract = await ethers.getContract('Land');
+  if (!LandContract) throw new Error('Land contract not found');
+  const admin = await LandContract.callStatic.getAdmin();
+  console.log(`land admin is: ${admin}`);
+
+  // set contract admin as minter and verify
+  await execute('Land', {from: admin}, 'setMinter', admin, true);
+  const isMinter = await LandContract.callStatic.isMinter(admin);
+  if (!isMinter)
+    throw new Error('admin is not minter even after calling setMinter');
```

Comment:
Is there a case where this happens? Else, we could move the execute here inside the if, and maybe after minting reset the admin isMinter state to the previous one

Reply:
done. thanks!